lib/iris/util.py

```diff
@@ -20,7 +20,8 @@
 """
 
 from __future__ import (absolute_import, division, print_function)
-from six.moves import (filter, input, map, range, zip)  # noqa
```

Comment:
We are using these two lines as boilerplate for all our modules to help with Python 3 compatibility. (you'll find them at the top of all files.)
I think it would be best to keep all of them (even if they aren't being used) as otherwise we may forget to include them.
In fact that is what the `#noqa` tag is there for - to stop flake8 complaining about unused imports

Reply:
I didn't even realise it wasn't there.  I'll pop it in.

Reply:
No idea how I removed that...